CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # Preamble ####################################################################
 #
-cmake_minimum_required(VERSION 3.17.0)
+cmake_minimum_required(VERSION 3.18.0)
```

Comment:
Technically, the `if(POLICY CMP0104)` can now be reduced because 3.18+ introduces that.

Want to re-check why does does not work with our logic in AMReX with CMake 3.20+ anyway, something must have been lost in translation... Will be a follow-up.